spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
+  describe 'GET #index' do
+    it 'returns a successful response' do
+      user = User.create(name: 'John Doe')
```

Comment:
- You have done great so far :+1: :+1: But there are a few requirements you still need to meet before moving to the next milestone :+1: :small_airplane: 

- - Screenshot reference:
![Selection_038](https://github.com/stino-x/Blog-Rails-/assets/107632780/5d3e9e06-8cd8-4bca-b5c3-8c7f2a33d3aa)

- [ ] As stated in the project requirement, after checking that the ` response status was correct.` and also that `a correct template was rendered.` You need to also check `If the response body includes the correct placeholder text. ` Currently, you do not have a check for that. At this current milestone, your view files don't need to contain any code logic, just a placeholder text. eg: the post index could have `<h2>Here is a list of all the posts from this user</h2>`. :smile:  Then there should be a test checking if the correct place holder was rendered. eg:
```ruby
    it 'Includes the correct placeholder text' do
      expect(response.body).to include('Here is a list of all the posts from this user')
    end
```

This is what the requirement says and you are expected to do this on the `Post index and Post show`. I'll suggest you implement this correctly. :smile: :waffle: